provider/terraform/custom_code.rb

```diff
@@ -87,8 +87,19 @@ class Examples < Api::Object
       # This list corresponds to the `get*FromEnv` methods in provider_test.go.
       attr_reader :test_env_vars
 
-      # the version of the example. Note that _all features_ used in an example
-      # must be set to the example min version.
+      # The version name of of the example's version if it's different than the
+      # resource version, eg. `beta`
+      #
+      # This should be the highest version of all the features used in the
+      # example; if there's a single beta field in an example, the example's
+      # min_version is beta. This is only needed if an example uses features
+      # with a different version than the resource; a beta resource's examples
+      # are all automatically versioned at beta.
+      #
+      # When an example has a version of beta, each resource must use the
+      # `google-beta` provider in the config and a `google-beta` block must be
```

Comment:
the part about the block doesn't seem to be true anymore, right?

Reply:
Yep! Found that out between submitting this and now haha. The test will pass w/o it now, but OiCS will be running Terraform `0.11` until `0.12` releases. Since OiCS still requires a block to be defined, I'd rather indicate that they're required here until then.